src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -226,10 +230,22 @@ def privacy_request_csv_download(privacy_request_query: Query) -> StreamingRespo
             "Request status",
             "Reviewer",
             "Time approved/denied",
+            "Denial reason",
         ]
     )
+    privacy_request_ids: List[str] = [r.id for r in privacy_request_query]
+    audit_log_query = db.query(AuditLog).filter(
```

Comment:
can we rename to `denial_audit_log_query` or something similar?

Reply:
Also, I like that you queried the db once to build a map of denial logs, instead of making separate queries for each privacy request.

Reply:
Thanks, and we can. Done!